src/vcdiff.cc

```diff
@@ -177,21 +178,20 @@ void VcdCtx::Process(const char* data, size_t len, bool is_last) {
 
 }
 
-bool VcdCtx::CheckError() {
+bool VcdCtx::CheckError(v8::Isolate* isolate) {
   if (!HasError())
     return true;
 
-  SendError();
+  SendError(isolate);
   return false;
 }
 
-void VcdCtx::SendError() {
-  v8::HandleScope scope;
+void VcdCtx::SendError(v8::Isolate* isolate) {
   v8::Local<v8::Value> args[2] = {
-    v8::String::New(GetErrorString(err_)),
-    v8::Number::New(static_cast<int>(err_)),
+    v8::String::NewFromUtf8(isolate, GetErrorString(err_)),
```

Comment:
As I stare, this does seem hack-ish. I had created a HandleScope elsewhere instead, but it didn't make much sense where I put it. I've moved it in commit 0fd2fbc723114fa27e44a8dbf06fc1b00cc0470f.